### PR TITLE
Add bulgarian

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

**Comment by Alir3z4** [comment]
Awesome.
This one seems it doesn't have `Bulgarian` in `languages.json` file.

File: languages.json [hunk 1]
```diff
@@ -8,6 +9,7 @@
 "fr": "french",
 "de": "german",
 "hu": "hungarian",
+"id": "indonesian",
```
Comment (norkans7) [inline languages.json]: I had forgotten Indonesian in the previous PR as well

**Comment by norkans7** [comment]
I think it is good now.

**Comment by Alir3z4** [comment]
@norkans7 Just merged in.
Great work.
Thanks

**Pull request merged**